File: Week_2/problem_9-Positions_in_array.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/D

n = int(input())
ar = list(map(int, input().split()))

for i in range(n):
    if ar[i] <= 10:
        print(f"A[{i}] = {ar[i]}")

File: Week_2/problem_9-Positions_in_array.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/D

#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int n;
    cin >> n;
    vector<int> ar(n);

    for (int i = 0; i < n; i++)
    {
        cin >> ar[i];

        if (ar[i] <= 10)
        {
            cout << "A[" << i << "] = " << ar[i] << endl;
        }
    }

    return 0;
}
